analysis/priest/src/TranslucentImage.tsx

```diff
@@ -0,0 +1,60 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import ConduitSpellText from 'parser/ui/ConduitSpellText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+import { TRANSLUCENT_IMAGE_INCREASE } from '../../priestdiscipline/src/constants';
+class TranslucentImage extends Analyzer {
+  damageReduced = 0;
+  damageDuringTranslucentImage = 0;
+  conduitRank = 0;
+  conduitIncrease = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.TRANSLUCENT_IMAGE.id);
+
+    if (!this.conduitRank) {
+      this.active = false;
+      return;
+    }
+    this.conduitIncrease = TRANSLUCENT_IMAGE_INCREASE[this.conduitRank];
```

Comment:
this should use `parser/core/conduitScaling` with the appropriate rank 1 value (which based on the other file is i think `6`?) unless the scaling doesn't follow regular conduit scaling (`rankOneValue * (1 + (rank - 1) * 0.1)`)